app/models/fee_calculation.rb

```diff
@@ -53,6 +53,10 @@ def from_planning_portal_data(pp_data)
     end
   end
 
+  def discount
+    total_fee.to_d - payable_fee.to_d
```

Comment:
We're converting `nil` to `0.0` by doing this - is that correct?

Reply:
I think so, just moved that logic from [this](https://github.com/unboxed/bops/pull/1723/files#diff-8ea26bef723ba324e88ffe97e985288c676e6e99dbce657d6361a28a3875a379L59) view to a model method.